models/dex/dex_sandwiched.sql

```diff
@@ -0,0 +1,64 @@
+{{ config(
+        
+        schema='dex',
+        alias = 'sandwiched',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['blockchain', 'tx_hash', 'project_contract_address', 'evt_index'],
```

Comment:
is `project_contract_address` needed here? that differs from all chain spells and the unique combo schema test

Reply:
ah forgot to remove, just did now, ty